test/framework/setup.go

```diff
@@ -25,18 +26,8 @@ type InstallFunc func(cluster Cluster) error
 var K8sScheme = runtime.NewScheme()
 
 func init() {
-	err := corev1.AddToScheme(K8sScheme)
-	if err != nil {
-		panic(err)
-	}
-	err = appsv1.AddToScheme(K8sScheme)
-	if err != nil {
-		panic(err)
-	}
-	err = kumav1alpha1.AddToScheme(K8sScheme)
-	if err != nil {
-		panic(err)
-	}
+	kube_utilruntime.Must(kube_clientgoscheme.AddToScheme(K8sScheme))
+	kube_utilruntime.Must(mesh_k8s.AddToScheme(K8sScheme))
```

Comment:
Suggest that we add  Kubernetes plugin util `NewScheme()` that returns a new scheme with everything registered in it already. Use that here, and also in the plugin boostrap.